apps/allocations/app/components/App/PeriodDetails.js

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { useAragonApi } from '../../api-react'
+import { Box, GU, Text, textStyle, useTheme } from '@aragon/ui'
+import throttle from 'lodash.throttle'
+import BigNumber from 'bignumber.js'
+import usePeriod from '../../hooks/usePeriod'
+import { ETHER_TOKEN_FAKE_ADDRESS, isTokenVerified } from '../../../../../shared/utils/token-utils'
+
+const MS_PER_DAY = 24 * 60 * 60 * 1000
+const MONTHS = [ 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec' ]
+const CONVERT_API_BASE = 'https://min-api.cryptocompare.com/data'
+const CONVERT_THROTTLE_TIME = 5000
```

Comment:
Every five seconds seems really frequent. Could we set this to once every, I don't know, minute?